prime_numbers.py

```diff
@@ -0,0 +1,26 @@
+# By listing the first six prime numbers: 2, 3, 5, 7, 11, and 13, we can see that the 6th prime is 13.
+
+# What is the 10,001st prime number?
+div = 10001
```

Comment:
two lines here

Reply:
you should also wrap all this in a function and then call it from `if __name__ == '__main__'`